pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -1144,8 +1143,23 @@ func (a *ApplicationBackupController) backupVolumes(backup *stork_api.Applicatio
 	}
 	// append skipped volumes
 	backup.Status.Volumes = append(backup.Status.Volumes, skipVolInfo...)
+	// add for fail and partial success
+	if len(backup.Status.Volumes) == len(skipVolInfo) {
+		// This case signifies that none of the volumes are successfully backed up
+		// hence marking it as failed
```

Comment:
We dont need this assignment here, just set `partialSuccess` and `partialFailed `accordingly`

Reply:
Made changes